src/styles/text/text.js

```diff
@@ -92,15 +92,32 @@ Object.assign(TextStyle, {
             return;
         }
 
-        q.feature = feature;
-        q.context = context;
-        q.layout.vertex = false; // vertex placement option not applicable to standalone labels
+        // text can be an array if a `left` or `right` orientation key is defined for the text source
+        // in which case, push both text sources to the queue
+        if (q instanceof Array){
```

Comment:
Nitpick but I think `Array.isArray()` was added (in ES5) deal with edge cases around `instanceof Array`. While maybe unlikely they would affect us, I feel like it's better to be consistent (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray#instanceof_vs_isArray).